app/index.js

```diff
@@ -154,6 +154,11 @@ module.exports = yeoman.generators.Base.extend({
     this.directory('.', '.');
   },
 
+  generateYoRc: function() {
+    this.log('Generating .yo-rc.json');
+    this.config.save();
+  },
+
```

Comment:
I see. Can you submit this change in a new commit please and include a unit-test for that? (A new pull request is ok too, so that this PR does not have to wait for that.)